### PR TITLE
Fix lambda deploy action regressions

### PR DESCRIPTION
**Pull request by robert-bo-davis**
Primary reviewer: @GPaoloni 

## Description
This fixes regressions introduced to the lambda deploy action by the move to npm workflows.

#261 should be merged before this.

### Verification steps
Run lambda deploy action for contact-retrieve-transcript job from bo-fix_lambda_action branch.

### CODE REVIEW

**Review by stephenhand — APPROVED** [review]

File: tsconfig.build.jobs.contact-retrieve-complete.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "./tsconfig.base.json",
+  "files": [],
+  "references": [
+    { "path": "packages/ssm-cache" },
+    { "path": "packages/s3-client" },
+    { "path": "packages/twilio-client" },
+    { "path": "jobs/contact-retrieve-recording-url" }
```
Comment (GPaoloni) [inline tsconfig.build.jobs.contact-retrieve-complete.json:8]: Is this reference correct?

File: tsconfig.build.jobs.contact-retrieve-recording-url.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "extends": "./tsconfig.base.json",
+  "files": [],
+  "references": [{ "path": "packages/ssm-cache" }, { "path": "jobs/contact-retrieve-complete" }]
```
Comment (GPaoloni) [inline tsconfig.build.jobs.contact-retrieve-recording-url.json:4]: Is this reference correct?

File: tsconfig.build.jobs.contact-retrieve-complete.json (hunk 1 above)
Comment (robert-bo-davis) [reply tsconfig.build.jobs.contact-retrieve-complete.json]: Oops. Good catch! Switched those around. Fixed.

**Comment by robert-bo-davis** [comment]
Oops. Meant to click the request new review from @GPaoloni, not @stephenhand.

**Review by stephenhand — APPROVED** [review]
LGTM

**Pull request merged**